infra/modules/worklytics-connector-specs/main.tf

```diff
@@ -759,7 +766,6 @@ EOT
         GRANT_TYPE : "refresh_token"
         REFRESH_ENDPOINT : "https://auth.atlassian.com/oauth/token"
         USE_SHARED_TOKEN : "TRUE"
-        ACCESS_TOKEN_CACHEABLE : "TRUE"
```

Comment:
Jira cloud has the `ACCESS_TOKEN` secret defined, so in theory it will not need that unless the secret is dropped...

Reply:
so I think we should do both, if possible; cache in the process even if storing in Secret Manager; otherwise will have roundtrip to secret manager on every request, which we shouldn't do unless we need to